app/src/main/java/chat/rocket/android/util/TokenSerialisableModel.kt

```diff
@@ -0,0 +1,36 @@
+package chat.rocket.android.util
+
+import java.io.Serializable
+
+class TokenSerialisableModel(
+    private val firstParam: String,
+    private val secondParam: String,
+    private val thirdParam: String = ""
+) :
+    Serializable {
+    companion object {
+        private val serialVersionUID: Long = 101L
+    }
+
+    private val tokenUserId: String
+    private val authToken: String
+    private val serverUrl: String
+
+    init {
+        tokenUserId = firstParam
+        authToken = secondParam
+        serverUrl = thirdParam
+    }
+
+    fun getFirst(): String {
+        return tokenUserId
+    }
+
+    fun getSecond(): String {
```

Comment:
Please replace to something more meaningful. Looking to the function name, programmers won't be able to know what this function returns until they look inside it and discovered that it returns an auth token.